pkg/clients/dynatrace/client.go

```diff
@@ -119,7 +112,6 @@ const (
 // Relevant token scopes
 const (
 	TokenScopeInstallerDownload     = "InstallerDownload"
-	TokenScopeDataExport            = "DataExport"
```

Comment:
I found mentions of this here https://github.com/Dynatrace/dynatrace-operator/blob/c1d3d3308b3fb8c52adbc03474a0b267ed0fc505/pkg/clients/dynatrace/token_test.go#L75-L76

however, that is a 5 year old test, that tests a mock... (updating it will give use nothing)

the dtclient package really needs a refactor...